build.py

```diff
@@ -0,0 +1,263 @@
+import argparse
+import json
+import os
+import pathlib
+import platform
+import shutil
+import subprocess
+import sys
+import urllib.parse
+import urllib.request
+import zipfile
+
+def ParseArguments():
+    parser = argparse.ArgumentParser ()
+    parser.add_argument ('--acVersion', dest = 'acVersion', type = str, required = True, help = 'Archicad version number list. Ex: "26 27"')
+    parser.add_argument ('--language', dest = 'language', type = str, required = False, help = 'Add-On language code list. Ex: "INT GER". Specify "ALL" for all languages in the configfile.' )
+    parser.add_argument ('--devKitPath', dest = 'devKitPath', type = str, required = False, help = 'Path to local APIDevKit')
+    parser.add_argument ('--release', dest = 'release', required = False, action='store_true', help = 'Build in localized Release mode.')
+    parser.add_argument ('--package', dest = 'package', required = False, action='store_true', help = 'Create zip archive.')
+    return parser.parse_args ()
+
+def ChangePermission(devKitFolder):

```

Comment:
The name of this function doesn't explain very well it's purpose. Could you please rename it to something like `AddRunPermissionToResConv`?

Reply:
Btw. why do we need this? Isn't this a problem in the DevKit creation process?

Reply:
Turns out we don't need it anymore. Removed.